Bottom: 51ef6ad1c26da2b22baa5d114c02de4cd91f104c
Top:    2a6fdc66c0a9ead1efd24a6a00f044feb5f584c6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 08:44:29 -0500

Refresh of all-docker_volume

---
diff --git a/docker/docker-mantisbt/mantisbt.sh b/docker/docker-mantisbt/mantisbt.sh
index 7c584d4..44dc2c3 100755
--- a/docker/docker-mantisbt/mantisbt.sh
+++ b/docker/docker-mantisbt/mantisbt.sh
@@ -56,8 +56,8 @@ done
 # make certian the containers exist
 docker inspect ${MANTISBT_CONTAINER_NAME} > /dev/null
 docker inspect ${MANTISBT_DB_CONTAINER_NAME} > /dev/null
-docker inspect ${MANTISBT_DV_NAME} > /dev/null
-docker inspect ${MANTISBT_DB_DV_NAME} > /dev/null
+docker volume inspect ${MANTISBT_DV_NAME} > /dev/null
+docker volume inspect ${MANTISBT_DB_DV_NAME} > /dev/null
 
 get_db_user_and_password() {
     db_user="$(docker 2>&1 exec ${MANTISBT_CONTAINER_NAME} grep -e '^$g_db_username' config_inc.php|sed 's|^.* = \"\(.*\)\";$|\1|' || true)"
